### PR TITLE
feat: CHAPI - store VP, get VP, get VP, Web Cred Query

### LINKED ISSUE #124: CHAPI - Get VP, Get VC, Store VP & Web Credential Queries

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- added store VP feature
- added get VP feature
- added get VC feature
- added web credential query support
- interop fixes
- Closes #124

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

File: cmd/user-agent/src/pages/GetVC.vue [hunk 1]
```diff
@@ -276,10 +417,59 @@ SPDX-License-Identifier: Apache-2.0
...
+                let data
+                if (creds.vcs) {
+                    await this.aries.verifiable.generatePresentation({
+                        verifiableCredential: creds.vcs,
+                        did: this.issuers[this.selectedIssuer].key,
+                        domain: this.domain,
+                        challenge: this.challenge,
+                        // TODO can be an option in view
+                        skipVerify: true,
+                        signatureType:"JsonWebSignature2020",
```
Comment (troyronda) [inline cmd/user-agent/src/pages/GetVC.vue:441]: why is this hardcoded?

Comment (sudeshrshetty) [reply cmd/user-agent/src/pages/GetVC.vue:441]: @fqutishat ^^
It is been added in previous commit, aries doesn't choose defaul anymoret, how can we make it as an option in CHAPI window?
@troyronda

Comment (fqutishat) [reply cmd/user-agent/src/pages/GetVC.vue:441]: @sudeshrshetty no need

Comment (fqutishat) [reply cmd/user-agent/src/pages/GetVC.vue:441]: I'm fixing that

**Pull request merged**